### DISCUSSION TITLE
Installation of carla simulator

### DISCUSSION THREAD

**Discussion by akshay-t19** [carla-simulator/carla#7619] [Q&A]
I installed packages from github. Afterwards i extracted them using tar command on my ubuntu system. After executing ./CarlaUE4.sh ,one window opened but it is blank and I cannot see anything? Is there any issue with my installation?
please advice

**Comment by Blyron** [comment]
Follow this guide 
https://carla.readthedocs.io/en/latest/start_quickstart/

**Reply by akshay-t19** [reply]
I have followed the same steps. I have a doubt that is it required to have 6gb dedicated gpu? I dont have dedicated GPU so is that a reason that causing issues?

**Reply by Blyron** [reply]
Yeah. Carla requires a GPU to run

**Reply by akshay-t19** [reply]
Okay thanks for your support
